comit/src/asset/bitcoin.rs

```diff
@@ -36,6 +37,54 @@ impl fmt::Display for Bitcoin {
     }
 }
 
+impl Serialize for Bitcoin {
```

Comment:
This ser/deser to/from decimal strings. We were doing it as satoshis which is not user friendly.
If you specify a btc quantity of "123.456" it will convert it to 12_345_600_000 satoshi and vice versa.

Reply:
Why are satoshis not user-friendly? They can be formatted into BTC by the UI if it wishes to. On the API level, being unambiguous and correct is more important than values being formatted for human readability :)

Also, we already have a module that formats a Bitcoin asset in a satoshi string. If we were to add another serialization form, it should be added as another module :)

I should have commented that at first already, sorry for only coming up with this only now :grimacing: 
Can we remove this decimal string stuff again in favor of satoshis please?